### PR TITLE
CAROTAN COLONIZATION: Even more Bunny outfits.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

image
I SPENT LITERAL HOURS PORTING THESE SEND HELP.

## Why It's Good For The Game
More bunny. More good. Now everyone can be bunny. I spent literal hours on this shit I can feel my fingers giving out

## Proof Of Testing

image

## Changelog

:cl: DimWhat for spriting, SigmaPredator for helping me with organizing this, and my (theselfish) poor keyboard.
add: Nearly every job has a Bunny outfit. I am not joking. Check your vendors.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/clothing/outfits/bunny.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	name = "Syndicate Bunny Assassin"
 	uniform = /obj/item/clothing/under/syndicate/syndibunny
 	suit = /obj/item/clothing/suit/jacket/tailcoat/syndicate
-	shoes = /obj/item/clothing/shoes/high_heels
+	shoes = /obj/item/storage/box/syndibunny
```
Comment (theselfish) [inline modular_zubbers/code/modules/clothing/outfits/bunny.dm:33]:
```suggestion
	shoes = /obj/item/clothing/shoes/fancy_heels/syndi
```

**Comment by Majkl-J** [comment]
What did you take and can I have some

**Review by Majkl-J — CHANGES_REQUESTED** [review]
This needs some credit comments to either the original PR, or spriter. Up to you how to do it, can do it individually for each item, or with BEGIN-END comments

**Comment by theselfish** [comment]
> This needs some credit comments to either the original PR, or spriter. Up to you how to do it, can do it individually for each item, or with BEGIN-END comments

The spriter is credited in the changelog, but sure.

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 2]
```diff
@@ -172,3 +174,562 @@
...
+	name = "security tailcoat"
+	desc = "A reinforced tailcoat worn by bunny themed security officers. Uses the same lightweight armor as the MK 1 vest, though obviously has lighter protection in the chest area."
+	icon_state = "sec"
+	inhand_icon_state = "armor"
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	dog_fashion = null
+	armor_type = /datum/armor/security_tailcoat
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:450]:
```suggestion
	armor_type = /datum/armor/suit_armor
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 3]
```diff
@@ -172,3 +174,562 @@
...
+/datum/armor/security_tailcoat
+	melee = 30
+	bullet = 30
+	laser = 30
+	energy = 40
+	bomb = 25
+	fire = 70
+	acid = 90
+	wound = 10
+
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:461]:
```suggestion
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 4]
```diff
@@ -172,3 +174,562 @@
...
+/obj/item/clothing/suit/armor/security_assistant_tailcoat
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:462]: subtype this under `/obj/item/clothing/suit/armor/security_tailcoat/assistant`

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 5]
```diff
@@ -172,3 +174,562 @@
...
+	name = "security assistant's tailcoat"
+	desc = "A reinforced tailcoat worn by bunny themed security assistants. The duller color scheme denotes a lower rank on the chain of bunny command."
+	icon_state = "sec_assistant"
+	inhand_icon_state = "armor"
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	dog_fashion = null
+	armor_type = /datum/armor/security_tailcoat
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:472]: Not needed if above is adressed
```suggestion
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 6]
```diff
@@ -172,3 +174,562 @@
...
+	name = "warden's tailcoat"
+	desc = "A reinforced tailcoat worn by bunny themed wardens. Stylishly holds hidden flak plates."
+	icon_state = "warden"
+	inhand_icon_state = "armor"
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	dog_fashion = null
+	armor_type = /datum/armor/warden_tailcoat
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:484]:
```suggestion
	armor_type = /datum/armor/suit_armor
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 7]
```diff
@@ -172,3 +174,562 @@
...
+/datum/armor/warden_tailcoat
+	melee = 30
+	bullet = 35
+	laser = 30
+	energy = 25
+	bomb = 20
+	fire = 50
+	acid = 50
+	wound = 10
+
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:495]:
```suggestion
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 8]
```diff
@@ -172,3 +174,562 @@
...
+	name = "head of security's tailcoat"
+	desc = "A reinforced tailcoat worn by bunny themed security commanders. Enhanced with a special alloy for some extra protection and style."
+	icon_state = "hos"
+	inhand_icon_state = "armor"
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	dog_fashion = null
+	armor_type = /datum/armor/hos_tailcoat
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:529]:
```suggestion
	armor_type = /datum/armor/armor_hos
```

File: modular_zubbers/code/modules/clothing/suits/jacket.dm [hunk 9]
```diff
@@ -172,3 +174,562 @@
...
+/datum/armor/hos_tailcoat
+	melee = 30
+	bullet = 30
+	laser = 30
+	energy = 40
+	bomb = 25
+	fire = 70
+	acid = 90
+	wound = 10
+
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/suits/jacket.dm:541]: This one is literally the same as /datum/armor/armor_hos 
```suggestion
```

File: modular_zubbers/code/modules/clothing/under/misc.dm [hunk 10]
```diff
@@ -94,3 +98,646 @@
...
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+
+/obj/item/clothing/under/rank/civilian/clown_bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/under/misc.dm:591]: I'm not sure how this works with the parent already calling create_storage, I'm guessing it works fine in this case? (Pls test)

**Review by Majkl-J — CHANGES_REQUESTED** [review]
A few small nitpicks

**Review by Majkl-J — CHANGES_REQUESTED** [review]
A few small nitpicks

File: modular_zubbers/code/modules/clothing/head/helmet.dm [hunk 11]
```diff
@@ -120,3 +120,502 @@
...
+
+//CAPTAIN
+
+/obj/item/clothing/head/hats/caphat/bunnyears_captain
+	name = "captain's bunny ears"
+	desc = "A pair of dark blue bunny ears attached to a headband. Worn in lieu of the more traditional bicorn hat."
+	icon_state = "captain"
+	inhand_icon_state = "that"
+	icon = 'modular_zubbers/icons/obj/clothing/head/hats.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/head/bunnyears.dmi'
```
[the remark this answers is not in the document]
Comment (theselfish) [reply modular_zubbers/code/modules/clothing/head/helmet.dm:132]: This one isn't needless as it inherets the properties of the captain hat.

File: modular_zubbers/code/modules/clothing/under/misc.dm (hunk 10 above)
Comment (theselfish) [reply modular_zubbers/code/modules/clothing/under/misc.dm:591]: Whoops, forgot to change that when I made it inheret the clown stuff.

**Review by Majkl-J — APPROVED** [review]

**Comment by aKromatopzia** [comment]
do these actually protect anything, if not based

**Comment by Majkl-J** [comment]
> do these actually protect anything, if not based

The sec ones have standard armor like any other sec item. Considering they don't cover legs/arms, which some options do, it's fine

**Pull request merged**